src/pages/lessons/[slug].tsx

```diff
@@ -4,7 +4,7 @@ import React, {FunctionComponent} from 'react'
 import Link from 'next/link'
 import {useRouter} from 'next/router'
 import EggheadPlayer from 'components/EggheadPlayer'
-import get from 'lodash/get'
+import {isEmpty, get} from 'lodash'
```

Comment:
I was curious which way is “better” for Lodash imports and it turns out the way you’ve used is recommended

https://lodash.com/per-method-packages